stdlib/3.3/xml/etree/ElementTree.pyi

```diff
@@ -80,6 +80,7 @@ class _ListDataStream(io.BufferedIOBase):
     def __init__(self, lst) -> None: ...
     def writable(self) -> bool: ...
     def seekable(self) -> bool: ...
+    @no_type_check  # not same signature as parent
     def write(self, b: str) -> None: ...
```

Comment:
AFAICT from some experimentation, write() actually takes bytes too. But since this is an internal class used only by tostringlist() maybe it's better to get rid of it?